README.md

```diff
@@ -30,18 +32,41 @@ bin/setup
 bin/dev
 ```
 
-5. The app should be now available at http://localhost:3000
+5. The app should be now available at <http://localhost:3000>
 
 ```bash
 open http://localhost:3000
 ```
 
+#### Running linters locally
+
+To run ERB linters with the provided configuration, follow these steps:
+
+1. Open your project's root directory in the terminal.
+2. Run the linting command:
+
+ ```bash
+ erb-lint
```

Comment:
I added a `bin/lint` script which makes it easier to remember, since you don't have to remember all the arguments and also don't need to remember to run erblint and standardb.

Can we simplify this to just:

```suggestion
bin/lint
```